tests/e2e/scenarios/test_reborn_webui_v2_streaming_run_control_api.py

```
@@ -570,36 +570,77 @@ async def test_reborn_v2_sse_reconnect_resumes_without_gap_or_duplicate_served(
         assert _latest_projection_items(replayed_cursor_events) == expected_items
 
 
-async def test_reborn_v2_websocket_origin_projection_and_shared_capacity_served(
+async def _read_session_frame(response, deadline_s: float = 45.0):
```

Ruff (0.16.3)

[warning] Missing return type annotation for private function `_read_session_frame`

(ANN202)